docs/insights/style.md

```diff
@@ -0,0 +1,438 @@
+
+This sniff ensures method names are defined using camel case.
+
+**Insight Class**: `PHP_CodeSniffer\Standards\PSR1\Sniffs\Methods\CamelCapsMethodNameSniff`
+
+## Else If declaration <Badge text="^1.0"/> <Badge text="Style" type="warn"/>
+
+This sniff verifies that there are no else if statements (elseif should be used instead).
+
+**Insight Class**: `PHP_CodeSniffer\Standards\PSR2\Sniffs\ControlStructures\ElseIfDeclarationSniff`
+
+## Switch declaration <Badge text="^1.0"/> <Badge text="Style" type="warn"/> <Badge text="configurable"/>
+
+This sniff ensures all switch statements are defined correctly.
+
+**Insight Class**: `PHP_CodeSniffer\Standards\PSR2\Sniffs\ControlStructures\SwitchDeclarationSniff`
+
+<details>
+    <summary>Configuration</summary>
+
+```php
+\PHP_CodeSniffer\Standards\PSR2\Sniffs\ControlStructures\SwitchDeclarationSniff::class => [
+    'indent' => 4,
+]
+```
+</details>
+
+## Upper case constant name <Badge text="^1.0"/> <Badge text="Style" type="warn"/>
+
+This sniff ensures that constant names are all uppercase.
+
+**Insight Class**: `PHP_CodeSniffer\Standards\Generic\Sniffs\NamingConventions\UpperCaseConstantNameSniff`
+
+## Alphabetically sorted uses <Badge text="^1.0"/> <Badge text="Style" type="warn"/>
+
+This sniff checks whether uses at the top of a file are alphabetically sorted.
+Follows natural sorting and takes edge cases with special symbols into consideration.
+
+**Insight Class**: `SlevomatCodingStandard\Sniffs\Namespaces\AlphabeticallySortedUsesSniff`
+
+## Namespace Spacing <Badge text="^1.0"/> <Badge text="Style" type="warn"/> <Badge text="configurable"/>
+
+This sniff enforces configurable number of lines before and after `namespace`.
+
+**Insight Class**: `SlevomatCodingStandard\Sniffs\Namespaces\NamespaceSpacingSniff`
+
+<details>
+    <summary>Configuration</summary>
+
+```php
+\SlevomatCodingStandard\Sniffs\Namespaces\NamespaceSpacingSniff::class => [
+    'linesCountBeforeNamespace' => 1,
+    'linesCountAfterNamespace' => 1,
+]
+```
+</details>
+
+## Require one namespace in file <Badge text="^1.0"/> <Badge text="Style" type="warn"/>
+
+This sniff checks there is only one namespace in a file.
+
+**Insight Class**: `SlevomatCodingStandard\Sniffs\Namespaces\RequireOneNamespaceInFileSniff`
+
+## Unused uses <Badge text="^1.0"/> <Badge text="Style" type="warn"/> <Badge text="configurable"/>
+
+This sniff detects unused `use` in a file.
+
+**Insight Class**: `SlevomatCodingStandard\Sniffs\Namespaces\UnusedUsesSniff`
+
+<details>
+    <summary>Configuration</summary>
+
+```php
+\SlevomatCodingStandard\Sniffs\Namespaces\UnusedUsesSniff::class => [
+    'searchAnnotations' => false,
+    'ignoredAnnotationNames' => [], // case sensitive list of annotation names that the sniff should ignore (only the name is ignored, annotation content is still searched). Useful for name collisions like @testCase annotation and TestCase class.
```

Comment:
I meant exactly what you have done here actually. Where you added a comment on what the property meant :)